examples/spark/pyspark_feature_catalog/features.py

```diff
@@ -2,28 +2,28 @@
 from pyspark.sql import functions as sf
 from with_columns import darkshore_flag, durotar_flag
 
-from hamilton.function_modifiers import tag
+from hamilton.function_modifiers.metadata import schema
 from hamilton.plugins.h_spark import with_columns
 
-WORLD_OF_WARCRAFT__SCHEMA = "zone:str, level:int, avatarId:int"
+WORLD_OF_WARCRAFT_SCHEMA = {"zone": str, "level": int, "avatarId": int}
```

Comment:
Why the need to constrain to python types? 

E.g. someone should be able to specify things like:
```python
"zone:str, level:int64, avatarId:int32"
```

I mean, this should be one of the possible ways to specify it, but I think as the only way, it's too restrictive.

Reply:
Sure, we can always add more. And `int32` versus `int64` are generally specific to certain frameworks (we wouldn't, likely, have this in, say, pydantic). Hence `int` as the common one. This was also the goal to have the different flavors, which map to each other in certain ways.